RecoTracker/IterativeTracking/python/InitialStep_cff.py

```diff
@@ -290,14 +293,19 @@
 initialStep = ClassifierMerger.clone()
 initialStep.inputClassifiers=['initialStepClassifier1','initialStepClassifier2','initialStepClassifier3']
 
-#LWTNN selector
+trackingPhase1.toReplaceWith(initialStep, TrackMVAClassifierPrompt.clone(
+     mva = dict(GBRForestLabel = 'MVASelectorInitialStep_Phase1'),
+     src = 'initialStepTracks',
+     qualityCuts = [-0.95,-0.85,-0.75]
```

Comment:
the baseline cloned from initialStepClassifier1; please revert to be consistent with pp_on_AA below

Reply:
Reverted back to cloning from initialStepClassifier1